README.md

```diff
@@ -23,23 +23,28 @@ pip install personal-graph
 ### Building a Working Memory for an AI
 
 ```python
-from personal_graph import Graph
+from personal_graph.graph import Graph
+
+your-litellm-client = LLMClient(client="llm-client", model_name="llm-model-name")
```

Comment:
The examples in the readme should mostly all use the default values (openai) instead of special clients. You can add a seperate section on how to use any model by providing an litellm client url etc

Reply:
Done